prepare/README.md

```diff
@@ -23,26 +23,26 @@ $ cargo run --release --bin system -- -r unidic-mecab-2.1.2_src -o system.dic
 
 This command requires the four files, `lex.csv`, `matrix.def`, `char.def`, and `unk.def`, to be in the directory specified by `-r`.
 
-## 2. Training mapping of connection ids
+## 2. Reordering mapping of connection ids
 
 Vibrato supports faster tokenization by improving the locality of reference through mapping connection ids.
 
-To tokenize sentences in `train.txt` and train the mapping,
+To tokenize sentences in `reorder.txt` and reorder the mapping,
```

Comment:
```suggestion
The reordering steps consist of
1. computing statistics from training data of sentences,
2. producing the reordered mapping, and
3. editing the system dictionary with the reordered mapping.

To produce the reordered mapping from sentences in `train.txt`,
```
I think `train.txt` is more appropriate to indicate the role.
Since the original documentation was not informative, I fixed it.